doc/examples/plot_entropy.py

```diff
@@ -3,17 +3,61 @@
 Entropy
 =======
 
-Image entropy is a quantity which is used to describe the amount of information
-coded in an image.
+In information theory, information entropy is the log-base-2 of the number of
+possible outcomes for a message.
+
+For an image, local entropy is related to the complexity contained in a given
+neighborhood, typically defined by a structuring element. A large number of
+various gray levels has a higher entropy than an homogeneous neighborhood.
+
+Entropy filter can detect subtle variations of local gray level distribution,
+in the example, the image is composed of two surfaces with two slightly
+different distributions.
+
+Image center has a random distribution in the range [-14,+14] centered on 128,
```

Comment:
Maybe something like: "A uniform random distribution in the range [-14, +14] in the middle of the image and a uniform random distribution in the range [-15, 15] at the image borders, both centered at a gray value of 128."